__exekutir__/roles/kommandir_workspace_setup/tasks/main.yml

```diff
@@ -123,8 +123,9 @@
   git:
     repo: "{{ item.repo }}"
     dest: "{{ kommandir_workspace }}/cache/{{ item.cachepath }}"
-    # Shallow clone is much faster, two commits will cover a merge-commit
-    depth: "{{ item.depth | default(2) }}"
+    # Shallow clone is much faster, two commits will cover a merge-commit and will be set per repo.
+    # default(omit) to workaround failure with python-bugzilla repo
+    depth: "{{ item.depth | default(omit) }}"
```

Comment:
Hmmm, maybe it could work.

```
$ ansible -i /dev/null -c local localhost -m debug -a 'msg="{{ omit if true else notexist | default(2) }}"'
```

Outputs "msg: Hello World" (which is the 'omit' behavior)